examples/3d/volumetric_fog.rs

```diff
@@ -60,6 +61,44 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
             ..default()
         });
 
+    // Add the point light
```

Comment:
It would be nice to add a UI option to cycle whether the lights are enabled, so that users can still see the raw effect.